src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,209 @@
+package duke;
+import duke.command.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Duke {
```

Comment:
I see that the iP follows coding standards and coding quality rather closely. Very good 👍

Reply:
Thanks for your comments in helping improve my code! I've made the amendments as necessary :))